components/director/hack/plugins/descriptionsdecorator/plugin.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/99designs/gqlgen/codegen/config"
 	"github.com/99designs/gqlgen/plugin"
 	"github.com/kyma-incubator/compass/components/director/hack/plugins"
-	"github.com/kyma-incubator/compass/components/director/pkg/log"
```

Comment:
Is this intentional?

Reply:
Log package was used only to print a debug statement, for which now we use "fmt". There was no actual benefit from it.